templates/tools/dockerfile/test/cxx_gcc_5_x64/Dockerfile.template

```diff
@@ -14,7 +14,7 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
-  FROM gcc:4.9
+  FROM gcc:5
```

Comment:
Shall we make this `gcc:5.1`? The `gcc:5` tag is currently pointed at version 5.5.0 https://hub.docker.com/layers/gcc/library/gcc/5/images/sha256-37139c6da088e06151617e741dc1dc62adefa3a4c9b9d28fc29ac83e629b4575?context=explore

Reply:
Yeah, I tried 5.1 but it's broken. It doesn't necessarily means that 5.1 gcc doesn't work but gcc docker image with gcc 5.1 isn't in a good shape.